portfolio/src/main/webapp/index.html

```diff
@@ -17,6 +17,7 @@
         <li class="nav-item projects">Projects</li>
         <li class="nav-item skills">Skills</li>
         <li class="nav-item cats">Cats</li>
+        <li class="nav-item shawarma">Shawarma</li>
```

Comment:
That sounds like a good day to review this CL considering what I've had for lunch:

![Shawarma](https://i.imgur.com/YtMYzTX.jpg)